src/components/landing-page/landing-page.tsx

```diff
@@ -1,22 +1,33 @@
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 import { JoinProduction } from "./join-production.tsx";
 import { CreateProduction } from "./create-production.tsx";
 import { ProductionsListContainer } from "./productions-list-container.tsx";
 import { useNavigateToProduction } from "./use-navigate-to-production.ts";
 import { DisplayContainer, FlexContainer } from "../generic-components.ts";
 import { useGlobalState } from "../../global-state/context-provider.tsx";
 import { isMobile } from "../../bowser.ts";
+import { TJoinProductionOptions } from "../production-line/types.ts";
 
 export const LandingPage = ({ setApiError }: { setApiError: () => void }) => {
-  const [{ joinProductionOptions, apiError }] = useGlobalState();
+  const [singleJoinProductionOptions, setSingleJoinProductionOptions] =
+    useState<TJoinProductionOptions | null>(null);
+  const [{ calls, apiError }] = useGlobalState();
+
+  useEffect(() => {
+    Object.entries(calls).forEach(([callId, callState]) => {
+      if (callId) {
+        setSingleJoinProductionOptions(callState.joinProductionOptions);
+      }
+    });
+  }, [calls]);
```

Comment:
I'm not sure what this is intended to do, it will loop through all call IDs and update `singleJoinProductionOptions` as many times as there are calls. This seems to only result in one update, since `singleJoinProductionOptions` will end up being the last value looped over